### PR TITLE
[WebApp] Profile: AccountSecurity section - added

### PR DESCRIPTION
**Pull request by vitto-moz**
https://www.notion.so/saladtech/FE-Web-app-Profile-Page-add-a-new-section-called-Account-Security-UI-only-b214c20882a54198acfaf27b4540c535?pvs=4

### CODE REVIEW

**Review by Maks19 — APPROVED** [review]

File: packages/web-app/src/modules/account-views/account-views/components/Account.tsx [hunk 1]
```diff
@@ -303,6 +304,7 @@ const _Account: FC<Props> = ({
               </div>
             </div>
           </div>
+          <AccountSecurity />
```
Comment (Maks19) [inline packages/web-app/src/modules/account-views/account-views/components/Account.tsx:307]: Let's feature flag it to keep master branch "clean"

File: packages/web-app/src/modules/account-views/account-views/components/AccountSecurity.tsx [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+              leadingIcon={<FontAwesomeIcon icon={faKey} className={classes.addPasskeyIcon} />}
+            />
+          </div>
+          <div className={classes.passkeysList}></div>
+          {mockedPasskeys.map((passkey) => {
+            return (
+              <div className={classes.passkeysListItem}>
+                <Text variant="baseS">{passkey.name}</Text>
+                <Text variant="baseS">{moment(passkey.dateAdded).format('MMMM DD, YYYY')}</Text>
+                <FontAwesomeIcon icon={faTrashCan} className={classes.deletePasskeyIcon} />
```
Comment (Maks19) [inline packages/web-app/src/modules/account-views/account-views/components/AccountSecurity.tsx:113]: Do you plan to add a conditional when to render it later?
image

Comment (vitto-moz) [reply packages/web-app/src/modules/account-views/account-views/components/AccountSecurity.tsx:113]: yes, I think we can add it during the logic implementation

File: packages/web-app/src/modules/account-views/account-views/components/Account.tsx (hunk 1 above)
Comment (vitto-moz) [reply packages/web-app/src/modules/account-views/account-views/components/Account.tsx]: temporarily conditional render - added

**Review by Maks19 — APPROVED** [review]

**Pull request merged**